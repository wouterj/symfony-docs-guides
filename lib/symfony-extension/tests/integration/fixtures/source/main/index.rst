Some Test Docs!
===============

.. image:: /_images/symfony-logo.png

.. toctree::
    :maxdepth: 1

    datetime
    form/form_type

.. _reference-forms-type-date-format:

A header
--------

Some info...
