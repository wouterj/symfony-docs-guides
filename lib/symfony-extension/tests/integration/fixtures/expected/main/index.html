<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8" />
    </head>
    <body>
        <div class="section" id="some-test-docs">
            <h1><a class="headerlink" href="#some-test-docs" title="Permalink to this headline">Some Test Docs!</a></h1>
            <img src="/_images/symfony-logo.png" />
            <div class="toctree-wrapper toc-size-md">
                <ul class="toctree toctree-level-1 toctree-length-2">
                    <li>
                        <a href="/datetime.html">DateTimeType Field</a>
                    </li>
                    <li>
                        <a href="/form/form_type.html">FormType Documentation</a>
                    </li>
                </ul>
            </div>
            <div class="section" id="a-header">
                <a id="reference-forms-type-date-format"></a>
                <h2><a class="headerlink" href="#a-header" title="Permalink to this headline">A header</a></h2>
                <p>Some info...</p>
            </div>
        </div>
    </body>
</html>
